proposals/simd/BinarySIMD.md

```diff
@@ -30,6 +30,7 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
 | `v128.load`                |    `0x00`| m:memarg           |
 | `v128.store`               |    `0x01`| m:memarg           |
 | `v128.const`               |    `0x02`| i:ImmByte[16]      |
+| `v128.andnot`              |    `0x03`| -                  |
```

Comment:
Could you move this down to the end of the opcode space instead of here? This belongs with the other logical operations, but there isn't a good spot for it right now. It's what we've tried to do with other new opcodes when it doesn't fit into the current opcode space.